docs/backend/schemas.md

```diff
@@ -217,6 +217,148 @@ Note this won't have behavior fields added to it at this stage, only the fields
 - {doc}`reference list of fields used in Plone </backend/fields>`
 
 
+(backend-schemas-directives-label)=
+
+## Schema directives
+
+With `plone.autoform` and `plone.supermodel`, we can use directives to add information to the schema fields.
+
+
+### Omit fields
+
+A field can be omitted entirely from all forms, or from some forms, using the `omitted` and `no_omit` directives.
+In this example, the `dummy` field is omitted from all forms, and the `edit_only` field is omitted from all forms except those that provide the `IEditForm` interface:
+
+```{code-block} python
+:emphasize-lines: 7,12,13
+:linenos:
+
+from z3c.form.interfaces import IEditForm
+from plone.supermodel import model
+from plone.autoform import directives as form
+
+class IMySchema(model.Schema):
+
+    form.omitted("dummy")
+    dummy = schema.Text(
+        title="Dummy"
+        )
+
+    form.omitted("edit_only")
+    form.no_omit(IEditForm, "edit_only")
+    edit_only = schema.TextLine(
+        title = "Only included on edit forms",
+        )
+```
+
+In supermodel XML, this can be specified as:
+
+```{code-block} xml
+:emphasize-lines: 3,9
+
+<field type="zope.schema.TextLine"
+        name="dummy"
+        form:omitted="true">
+    <title>Dummy</title>
+</field>
+
+<field type="zope.schema.TextLine"
+        name="edit-only"
+        form:omitted="z3c.form.interfaces.IForm:true z3c.form.interfaces.IEditForm:false">
+    <title>Only included on edit form</title>
+</field>
+```
+
+`form:omitted` may be either a single boolean value, or a space-separated list of `<form_interface>:<boolean>` pairs.
+
+
+### Reorder fields
+
+A field's position in the form can be influenced using the `order_before` and `order_after` directives.
+In this example, the `not_last` field is placed before the `summary` field, even though it is defined afterward:
+
+```{code-block} python
+:emphasize-lines: 12
+:linenos:
+
+from plone.supermodel import model
+from plone.autoform import directives as form
+
+class IMySchema(model.Schema):
+
+    summary = schema.Text(
+        title="Summary",
+        description="Summary of the body",
+        readonly=True
+        )
+
+    form.order_before(not_last="summary")
+    not_last = schema.TextLine(
+        title="Not last",
+        )
+```
+
+The value passed to the directive may be either `*`, indicating before or after all fields, or the name of another field.
+Use `<.fieldname>` to refer to the field in the current schema or a base schema.
```

Comment:
is the dot by intend?

Reply:
If it is intentional, then it should be formatted as:

`.<fieldname>` or `.<field_name>`

...and perhaps using snake_case is more legible?

Reply:
I've fixed that